pkg/operator/controller/status.go

```diff
@@ -139,6 +139,11 @@ func computeOperatorStatusConditions(conditions []configv1.ClusterOperatorStatus
 			}
 		}
 	}
+	if numIngresses == 0 {
```

Comment:
Not sure I understand the intent here. Internally misrepresenting the actual number of ingresses seems weird.